serverless.yml

```diff
@@ -75,3 +64,120 @@ functions:
           method: post
           cors: true
           authorizer: aws_iam
+  testCognito:
+    handler: handler.test1
+    events:
+      - http:
+          path: test1
+          method: get
+          cors: true
+          authorizer: aws_iam
+resources:
+  Resources:
```

Comment:
This can be extracted into its own file, so we don't pollute the file. It can be done like this
```
# Create our resources with separate CloudFormation templates
resources:
  # API Gateway Errors
  - ${file(resources/api-gateway-errors.yml)}
```

Reply:
done